src/app/_components/comment-card/comment-card.ts

```diff
@@ -0,0 +1,19 @@
+import { Input, Component, ViewChild } from '@angular/core';
+import { ContentService } from '_services/index';
+import { ApiService } from '_services/api.service';
+
+@Component({
+  selector: 'comment-card',
+  templateUrl: 'comment-card.html',
+  styleUrls: ['comment-card.scss']
+})
+export class CommentCardComponent {
+  @Input('comment')
+  public comment: any;
+
+  constructor (
+    public content: ContentService,
```

Comment:
I see you've included the contentService but not used it. ContentService.body() can be used to print content and maintain line-breaks (converted to <br> elements). It should either be used or removed. Thanks!